pkg/providers/imagefamily/bootstrap/cse_cmd_nbcontract.sh.gtpl

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+set -o allexport # export all variables to subshells
+echo '#EOF' >> /opt/azure/manifest.json # wait_for_file looks for this
+mkdir -p /var/log/azure/Microsoft.Azure.Extensions.CustomScript/events # expected, but not created w/o CSE
+
+echo $(date),$(hostname) > /var/log/azure/cluster-provision-cse-output.log;
+for i in $(seq 1 1200); do
+grep -Fq "EOF" /opt/azure/containers/provision.sh && break;
+if [ $i -eq 1200 ]; then exit 100; else sleep 1; fi;
+done;
+{{if getIsAksCustomCloud .CustomCloudConfig}}
+for i in $(seq 1 1200); do
+grep -Fq "EOF" {{getInitAKSCustomCloudFilepath}} && break;
+if [ $i -eq 1200 ]; then exit 100; else sleep 1; fi;
+done;
+REPO_DEPOT_ENDPOINT="{{.CustomCloudConfig.GetRepoDepotEndpoint}}"
+{{getInitAKSCustomCloudFilepath}} >> /var/log/azure/cluster-provision.log 2>&1;
+{{end}}
+ADMINUSER={{getLinuxAdminUsername .GetLinuxAdminUsername}}
```

Comment:
Originally we default it to `azureuser`, but that logic got removed. Will it be defaulted later on? Otherwise let's add that back until we pass it from RP (I think I will do it)

Reply:
the getter will set it as "azureuser" if it's empty.
``` 
func getLinuxAdminUsername(username string) string {
	if username == "" {
		return nbcontractv1.DefaultLinuxUser
	}
	return username
}